src/components/PowerBIContainer/PowerBiContainer.tsx

```diff
@@ -16,26 +8,26 @@ const PowerBIContainer = ({
   panel: { fields: ReportData };
   pageName?: string;
 }) => {
-  const { macroTheme, title, source } = panel.fields;
+  const { macroTheme, title, source, date } = panel.fields;
+  const dateObj = date ? new Date(date) : null;
+  const formattedDate = dateObj ? dateObj.toLocaleDateString("pt-BR") : "";
 
   return (
-    <Wrapper>
-      <Header>
-        <Home href="/">
-          <RightIcon id="expand" size={10} />
-          <Text>Página inicial</Text>
-        </Home>
-        <Title>{macroTheme}</Title>
-      </Header>
+    <div className="flex flex-col w-max h-full z-0 bg-white gap-2 overflow-x-scroll p-4">
```

Comment:
```suggestion
    <div className="flex flex-col max-w-screen h-full z-0 bg-white gap-2 overflow-x-scroll p-4">
```

I think this might solve the issue Rodrigo pointed out